plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -198,13 +198,15 @@ def get_metadata(self):
         try:
             metadata_files.download_repomd()
         except IOError, e:
-            raise FailedException(str(e))
+            raise PulpCodedException(error_code=error_codes.RPM1004, reason=str(e))
```

Comment:
Should this get a `_logger.debug(traceback.format_exc())` statement like the one above? I'm not sure.